pkg/controller/controller.go

```diff
@@ -209,6 +213,15 @@ func (c *Controller) onAddActionSet(as *crv1alpha1.ActionSet) error {
 	if err := validate.ActionSet(as); err != nil {
 		return err
 	}
+	// ActionSet created
```

Comment:
This code might be better placed within c.handleActionSet (after determining status is pending, before updating status to be running).

Reply:
Done.